lib/policies/load-balancer/balancer.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const httpProxy = require('http-proxy');
+const logger = require('../../logger').gateway;
+
+const ConfigurationError = require('../../errors').ConfigurationError;
+const RoundRobin = require('./round-robin');
+
+const strategies = {
+  'round-robin': RoundRobin
+};
+
+const createStrategy = (name, proxy, endpoints) => {
+  const Strategy = strategies[name];
+  return new Strategy(proxy, endpoints);
+};
+
+module.exports = function (params, config) {
+  const endpoints = params.serviceEndpoints
+    .map(name => {
+      const endpoint = config.gatewayConfig.serviceEndpoints[name];
```

Comment:
I would say this should be only in a way @geek is suggesting. Because next time when we want to introduce consul\etcd support we will just define some `key` for service endpoint that will be used for lookup. this key will be resolved to  `urls` array.